spec/services/cclf/fee/misc_fee_adapter_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe CCLF::Fee::MiscFeeAdapter, type: :adapter do
+  let(:fee) { instance_double('fee') }
```

Comment:
**[not a blocker]** If you're not using an `instance_double` on an actual _class_, I would argue the use of it is not very effective (might as well use a normal `double`).